.github/workflows/external-contribution-labeler.yaml

```diff
@@ -18,10 +18,53 @@ jobs:
     permissions:
       pull-requests: write
     steps:
-      - name: Install Cilium CLI
+        # Detect if the secret 'CHECK_TEAM_ORG' is set. If it's not set, don't
+        # bother running this GH workflow.
+      - name: Check if CHECK_TEAM_ORG is set in github secrets
+        id: check_secret
         run: |
-          echo author_association=${{ github.event.pull_request.author_association }}
+          echo "is_CHECK_TEAM_ORG_set: ${{ secrets.CHECK_TEAM_ORG != '' }}"
+          echo is_CHECK_TEAM_ORG_set="${{ secrets.CHECK_TEAM_ORG != '' }}" >> $GITHUB_OUTPUT
+
+      - name: Get token
+        # Get a token with the read:org permissions so that the GH action
+        # can read the team membership for a user. We need to do this over a
+        # GH app because GH actions don't have support for these type of
+        # permissions.
```

Comment:
Meh. Hopefully we won't have any issue since you forked and properly own the code, but this is really a sketchy workaround.